crates/re_viewer/src/web.rs

```diff
@@ -80,6 +81,26 @@ impl WebHandle {
     pub fn panic_callstack(&self) -> Option<String> {
         self.runner.panic_summary().map(|s| s.callstack())
     }
+
+    #[wasm_bindgen]
+    pub fn add_receiver(&self, url: &str) {
+        let Some(mut app) = self.runner.app_mut::<crate::App>() else {
+            return;
+        };
+        let rx = url_to_receiver(url, app.re_ui.egui_ctx.clone());
+        app.add_receiver(rx);
+    }
+
+    #[wasm_bindgen]
+    pub fn remove_receiver(&self, url: &str) {
+        let Some(mut app) = self.runner.app_mut::<crate::App>() else {
+            return;
+        };
+        app.msg_receive_set().remove_by_uri(url);
+        if let Some(store_hub) = app.store_hub.as_mut() {
+            store_hub.remove_recording_by_uri(url);
+        }
```

Comment:
Is there any situation where we want to remove the receiver, but keep the recording that has been received so far? I'm imagining a case where a user is connected to a live stream and they want to stop getting updates but still inspect the data.

Reply:
Definitely. The APIs exposed here were made just to faciliate the React-style magic. I guess we could expose a separate `disconnect` API that just closes the stream, and preserves the recording.

Reply:
I'll leave it out of here for now, I don't know what that would look like on the React side